xlive/Blam/Engine/sound/sound_definitions.h

```diff
@@ -84,7 +42,7 @@ struct sound_definition_v1
 {
 	c_flags_no_init<e_sound_definition_flags, uint16, k_sound_definition_flag_count> flags;
 
-	e_sound_definition_class sound_class;
+	int8/*e_sound_class_type*/ sound_class;
```

Comment:
?

Reply:
e_sound_class_type is the enum and it's an int16, they use int8 in the definition for padding reasons

Reply:
You guys should add `c_enum` for this very reason